modules/tensor_mechanics/src/kernels/SolidMechInertialForceTensors.C

```diff
@@ -0,0 +1,53 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "SolidMechInertialForceTensors.h"
```

Comment:
Can't we just call this Kernel ```InertialForce```? Having an abbreviated _solid mechanics_ and a _tensors_ (even though it neither is part of the solid mechanics module, nor does the kernel compute any tensors) seems a little confusing.